File: auto-docs/test_executables/python/error-bar-asymmetric-array.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('theengineear', 'o9zlr0hy6z')

data = Data([
    Scatter(
        x=[1, 2, 3, 4],
        y=[2, 1, 3, 4],
        error_y=ErrorY(
            array=[0.1, 0.2, 0.1, 0.1],
            type='data',
            symmetric=False,
            arrayminus=[0.2, 0.4, 1, 0.2]
        )
    )
])

if 'fig' not in locals():
    if 'data' not in locals():
        raise Exception('no data OR figure!!')
    fig = dict(data=data)  # assumes fig or data
if 'layout' not in fig:
    fig['layout'] = dict()
if 'margin' not in fig['layout']:
    fig['layout']['margin'] = dict(t=50, b=50, r=50, l=50)
fig['layout'].update(autosize=False, width=500, height=500)


plot_url = py.plot(data, filename='error-bar-asymmetric-array', auto_open=False)

File: auto-docs/executables/node/error_bar_asymmetric_array.js
var plotly = require('plotly')('theengineear', 'o9zlr0hy6z')

var data = [
  {
    x: [1, 2, 3, 4], 
    y: [2, 1, 3, 4], 
    error_y: {
      array: [0.1, 0.2, 0.1, 0.1], 
      type: "data", 
      symmetric: false, 
      arrayminus: [0.2, 0.4, 1, 0.2]
    }, 
    type: "scatter"
  }
];

var graph_options = {filename: "error-bar-asymmetric-array", fileopt: "overwrite", auto_open: "false"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});